tests/integration/run_tests.rs

```diff
@@ -4560,15 +4560,14 @@ async fn websocket_server_multi_field_connection_header() {
   assert!(child.wait().unwrap().success());
 }
 
-// TODO(bartlomieju): this should use `deno run`, not `deno test`; but the
-// test hangs then. https://github.com/denoland/deno/issues/14283
 #[tokio::test]
 async fn websocket_server_idletimeout() {
+  test_util::timeout!(60);
```

Comment:
This test was using `deno test` to bypass a hang, but that stopped working with this change. I (mostly) fixed the test by just forcing an exit.